mlflow/server/js/src/components/ParallelCoordinatesPlotPanel.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ParallelCoordinatesPlotView from './ParallelCoordinatesPlotView';
+import { ParallelCoordinatesPlotControls } from './ParallelCoordinatesPlotControls';
+import { getSharedMetricKeysByRunUuids, getSharedParamKeysByRunUuids } from '../reducers/Reducers';
+import _ from 'lodash';
+
+import './ParallelCoordinatesPlotPanel.css';
+
+export class ParallelCoordinatesPlotPanel extends React.Component {
+  static propTypes = {
+    runUuids: PropTypes.arrayOf(String).isRequired,
+    sharedParamKeys: PropTypes.arrayOf(String).isRequired,
+    sharedMetricKeys: PropTypes.arrayOf(String).isRequired,
+  };
+
+  static defaultProps = {
```

Comment:
What is this for? Aren't these props always required?

Reply:
Yes, this can be removed now. It was added when at the beginning to avoid NPE.